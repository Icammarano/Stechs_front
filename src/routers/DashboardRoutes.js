import React from "react";
import { Route, Switch } from "react-router-dom";
import { HomeScreen } from "../components/home/HomeScreen";
import { Navbar } from "../components/ui/NavBar";
import { VendorScreen } from "../components/vendors/VendorScreen";
import { VendorsModelsScreen } from "../components/vendors/VendorsModelsScreen";

export const DashboardRoutes = () => {
    return (
        <>
            <Navbar />
            <div>
                <Switch>
                    <Route exact path="/vendors" component={VendorScreen} />
                    <Route
                        exact
                        path="/vendors/AddModels"
                        component={VendorsModelsScreen}
                    />
                    <Route path="/" component={HomeScreen} />
                </Switch>
            </div>
        </>
    );
};
