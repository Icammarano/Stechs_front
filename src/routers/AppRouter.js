import React from "react";
import { Route, BrowserRouter as Router, Switch } from "react-router-dom";
import { DashboardRoutes } from "./DashboardRoutes";

export const AppRouter = () => {
    return (
        <Router>
            <div>
                <Switch>
                    <Route path="/" component={DashboardRoutes} />
                </Switch>
            </div>
        </Router>
    );
};
