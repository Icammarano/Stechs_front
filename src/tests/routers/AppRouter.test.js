import { shallow } from "enzyme";
import { AppRouter } from "../../routers/AppRouter";

describe("Pruebas en <AppRouter />", () => {
    test("Debe mostrarse correctamente", () => {
        const wrapper = shallow(<AppRouter />);
        expect(wrapper).toMatchSnapshot();
    });
});
