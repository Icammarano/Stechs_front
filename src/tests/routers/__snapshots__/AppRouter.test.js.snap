// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <AppRouter /> Debe mostrarse correctamente 1`] = `
<BrowserRouter>
  <div>
    <Switch>
      <Route
        component={[Function]}
        path="/"
      />
    </Switch>
  </div>
</BrowserRouter>
`;
